public/configs/v1/env.json

```diff
@@ -315,7 +315,7 @@
             "CCTPDepositOnly": true,
             "isSlTpEnabled": true,
             "isSlTpLimitOrdersEnabled": false,
-            "isOhlcEnabled": false
+            "isOhlcEnabled": true
```

Comment:
is this flag used within Abacus? Otherwise we are probably able to deprecate?

Reply:
yessir (not used in abacus) - just wanted to have it in testnet / staging for a bit for PMs to play around with before cleaning up. Will clean up with the variable rename!